core/src/main/resources/reference.conf

```diff
@@ -217,10 +217,10 @@ cassandra-journal {
   target-partition-size = 500000
 
   # Maximum size of result set
-  max-result-size = 50001
+  max-result-size = 250
```

Comment:
That used to be pretty big :)

Reply:
I have no clue why it was that big, doesn't make sense too me. Of course it will be faster if you have the memory, but I don't think that is something we can assume.

Reply:
and a typical replay should be at most a few 100s events, otherwise snapshots should be used